specifyweb/frontend/js_src/lib/components/DataModel/businessRules.ts

```diff
@@ -469,6 +469,7 @@ export function attachBusinessRules(
 export type BusinessRuleResult<SCHEMA extends AnySchema = AnySchema> = {
   readonly localDuplicates?: RA<SpecifyResource<SCHEMA>>;
   readonly saveBlockerKey?: string;
+  readonly resource?: SpecifyResource<SCHEMA>;
```

Comment:
(optional)
I understand that this is needed for the case that a change occurs and a save blocker needs to be set on another resource other than the one attached to the businessrule manager running the rule. 

Can you add a comment here explaining that, or rename the variable to make its purpose a little clearer? 
Something like: 
```suggestion
  readonly resourceToSaveBlock?: SpecifyResource<SCHEMA>;
```

Reply:
added a comment